### PR TITLE
Add: Dependency downloading script

### PR DESCRIPTION
**Pull request by ramzzzay**
Added fsx script for dependency downloading for {Win,Linux,MacOS} OS

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ file, or invoke the following commands in developer console:
 ```console
 $ nuget restore
 $ msbuild
-$ powershell scripts/deps-windows.ps1 # consult the script documentation, there're additional parameters
+$ fsi scripts/deps-downloader.fsx win {Build_Type}(debug/release) # consult the script documentation, there're additional parameters
```
Comment (ForNeVeR) [inline Readme.md]: 1. The script should setup both Debug and Release configurations. Please remove the second argument here.
2. Remove the comment about the documentation; we have no other parameters or documentation.

File: Readme.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ You'll need [Mono][mono] and [NuGet][nuget] installed.
 ```console
 $ nuget restore
 $ xbuild
+$ fsi scripts/deps-downloader.fsx linux {Build_Type}(debug/release) # consult the script documentation, there're additional parameters
```
Comment (ForNeVeR) [inline Readme.md]: Same as above.

File: Readme.md [hunk 3]
```diff
@@ -50,6 +51,17 @@ $ xbuild
...
 
+#### MacOS
+
+You'll need [Mono][mono] and [NuGet][nuget] installed.
+
+Either use [JetBrains Rider][jetbrains-rider] to open and build `MoonBaseBuilder.sln`
+file, then invoke the following commands console:
+
+```console
+$ fsi scripts/deps-downloader.fsx macOS {Build_Type}(debug/release) # consult the script documentation, there're additional parameters
```
Comment (ForNeVeR) [inline Readme.md]: Same note as above (about the script).

File: Readme.md [hunk 4]
```diff
@@ -50,6 +51,17 @@ $ xbuild
 
 Requires NixOS 16.09+.
 
+#### MacOS
+
+You'll need [Mono][mono] and [NuGet][nuget] installed.
+
+Either use [JetBrains Rider][jetbrains-rider] to open and build `MoonBaseBuilder.sln`
+file, then invoke the following commands console:
```
Comment (ForNeVeR) [inline Readme.md]: "Currently, we recommend to build the project using JetBrains Rider. Open and build `MoonBaseBuilder.sln` file, then invoke the following command in the terminal:"

File: Readme.md [hunk 5]
```diff
@@ -50,6 +51,17 @@ $ xbuild
 
 Requires NixOS 16.09+.
 
+#### MacOS
```
Comment (ForNeVeR) [inline Readme.md]: It's macOS (first letter shouldn't be capital).

File: Readme.md [hunk 6]
```diff
@@ -67,6 +79,15 @@ $ cd MarsBaseBuilder/bin/Debug
 $ mono ./MarsBaseBuilder.exe
 ```
 
+On MacOS:
```
Comment (ForNeVeR) [inline Readme.md]: Please remove that section. Let's just say "On Linux or macOS:" above.

File: Readme.md [hunk 7]
```diff
@@ -105,6 +126,7 @@ Attribution 4.0 International License][cc-by-license].
 [nuget]: https://www.nuget.org/
 [travis]: https://travis-ci.org/ForNeVeR/1969
 [visual-studio]: https://www.visualstudio.com/
+[jetbrains-rider]: https://www.jetbrains.com/rider/
```
Comment (ForNeVeR) [inline Readme.md]: I keep the links section in alphabetical order. Please move that link to the proper position.

File: scripts/deps-downloader.fsx [hunk 8]
```diff
@@ -0,0 +1,18 @@
+let baseUrl = "https://github.com/MonoGame/MonoGame.Dependencies/raw/master/SDL"
+let pathToProjectBin = sprintf "../MarsBaseBuilder/bin/%s" fsi.CommandLineArgs.[2]
```
Comment (ForNeVeR) [inline scripts/deps-downloader.fsx]: That's not a good idea. Consider user running the script from the project root, or from some other directory.

Replace it with `"__SOURCE_DIRECTORY__/../MarsBaseBuilder/bin/%s"`.

Comment (ForNeVeR) [inline scripts/deps-downloader.fsx]: Also, please don't get the configuration from the command line, and instead process both `Release` and `Debug`.

File: scripts/deps-downloader.fsx [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+let bit = if System.Environment.Is64BitOperatingSystem then "x64" else "x86"
+let wc = new System.Net.WebClient()
+match fsi.CommandLineArgs.[1] with
+    | "macOS" ->
```
Comment (ForNeVeR) [inline scripts/deps-downloader.fsx]: Please indent this to the left (remove one indentation level from every line).

File: scripts/deps-downloader.fsx [hunk 10]
```diff
@@ -0,0 +1,18 @@
+let baseUrl = "https://github.com/MonoGame/MonoGame.Dependencies/raw/master/SDL"
+let pathToProjectBin = sprintf "../MarsBaseBuilder/bin/%s" fsi.CommandLineArgs.[2]
+let bit = if System.Environment.Is64BitOperatingSystem then "x64" else "x86"
+let wc = new System.Net.WebClient()
```
Comment (ForNeVeR) [inline scripts/deps-downloader.fsx]: `use` instead of `let` here!

**Comment by ramzzzay** [comment]
Fixed. Check please.

**Comment by ramzzzay** [comment]
Done. Check please.

**Pull request merged**